projects/step-core/src/lib/modules/table/shared/table-remote-data-source.ts

```diff
@@ -150,19 +150,22 @@ export class TableRemoteDataSource<T> implements TableDataSource<T> {
     filter((forceNavigate) => forceNavigate === true)
   );
 
-  private typeFilter?: { [key: string]: SearchValue };
+  private filters: { [key: string]: SearchValue } = {};
 
   constructor(
     private _tableId: string,
     private _rest: TableApiWrapperService,
     private _requestColumnsMap: { [key: string]: string },
-    private _typeFilter?: string[]
+    private _filters?: { [key: string]: string[] }
```

Comment:
personal preference - ```_filters?: Record<string, string[]>```

Reply:
I will add it for discussion to Trello, I don't have a preference but if we want to go with one of the two we should do it everywhere (i.e. also for "_requestColumnsMap" above this)